examples/object_detection/android-jetpack-compose/app/src/main/java/com/example/objectdetection/composables/TabsTopBar.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.objectdetection.composables
+
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import com.example.objectdetection.R
+
+// This composable is used in Home screen to navigate between camera view and gallery view
+
+// It takes an index to display an indication of which tab we're currently in, and a function
+// to set that index when a tab is tapped
+
+@Composable
+fun TabsTopBar(
+    selectedTabIndex: Int,
+    setSelectedTabIndex: (Int) -> Unit,
+) {
+    TabRow(selectedTabIndex = selectedTabIndex) {
+        Tab(
+            unselectedContentColor = Color.Gray,
+            selected = selectedTabIndex == 0,
+            onClick = {
+                setSelectedTabIndex(0)
+            },
+            text = { Text("Camera") },
+            icon = {
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_baseline_photo_camera_24),
+                    contentDescription = ""
+                )
+            },
+        )
+        Tab(
+            unselectedContentColor = Color.Gray,
+            selected = selectedTabIndex == 1,
+            onClick = {
+                setSelectedTabIndex(1)
+            },
+            text = { Text("Gallery") },
```

Comment:
Is it possible to turn this string into a string resource item (strings.xml)? Any constant strings in Kotlin classes should be thought about for migrating them into a file that can be localized or easily edited (what if someone comes along in a year and says "Android doesn't call it a Gallery anymore! Now it's a photo viewer")